e3sm_to_cmip/util.py

```diff
@@ -306,195 +359,389 @@ def print_var_info(handlers, freq=None, inpath=None, tables=None, outpath=None):
                 if not has_vars:
                     continue
                 messages.append(msg)
-    
-    
+
     if outpath is not None:
-        with open(outpath, 'w') as outstream:
+        with open(outpath, "w") as outstream:
             yaml.dump(messages, outstream)
     else:
         pprint(messages)
+
+
 # ------------------------------------------------------------------
 
-def get_table(table, variable, freq, tables):
+
+def _load_handlers(
+    handlers_path: str,
+    tables_path: str,
+    var_list: List[str],
+    freq: str = "mon",
+    realm: str = "atm",
+    simple: bool = False,
+) -> List[Dict[str, Any]]:
+    """Loads default and/or complex handlers based on a list of variable names.
+
+    If this function is called with `simple=True`, then the `freq` argument
```

Comment:
I think `simple=True` should have the same logic as `False`?

Reply:
This conditional was kept from the previous code:
https://github.com/E3SM-Project/e3sm_to_cmip/blob/cee5c228f6085b0afd4a4c4696f0c947b1514f50/e3sm_to_cmip/util.py#L393-L406

https://github.com/E3SM-Project/e3sm_to_cmip/blob/cee5c228f6085b0afd4a4c4696f0c947b1514f50/e3sm_to_cmip/util.py#L453-L467

I'll have to investigate any side-effects if we allow CMIP6 tables for other frequencies to be used if we want to remove `simple=True`.

Reply:
The check for `simple` is now removed so it behaves the same as normal runs.